### PR TITLE
Create Profile page

### PR DESCRIPTION
**Pull request by oliverhees**

### CODE REVIEW

File: components/ProfileForm/index.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    // Current Date
+    const currentDate = new Date();
+
+    // Save date in the desired format
+    setFormData({ ...formData, entryDate: formatDate(currentDate) });
+  }, []);
+
+  return (
+    <ProfileFormDiv onSubmit={handleSubmit}>
+      <div className="column-left">
```
Comment (HannesOster) [inline components/ProfileForm/index.js:35]: Here youre using css instead of styled components

Comment (HannesOster) [inline components/ProfileForm/index.js]: Here for example youre using css instead of styled-components

**Review by HannesOster — APPROVED** [review]
Hey Olli,

Well done!

The only thing i would criticize is that your using css sometimes instead of styled-components. 
I guess you will fix that later?

File: components/ProfileForm/index.js (hunk 1 above)
Comment (HannesOster) [reply components/ProfileForm/index.js]: Also im missing the vercel deployment, but for the next step you have to do this anyway ;)

Comment (oliverhees) [reply components/ProfileForm/index.js]: @HannesOster thanks for your reply! Problem by Vercel is that it render the index. Here i deploy the profile page, you can find it her in the deployment -> https://aipersonaltrainer-llzrkur8v-oliverhees.vercel.app/profile

**Pull request merged**